ui/src/app/cohort-review/detail-header/detail-header.component.tsx

```diff
@@ -399,8 +411,8 @@ export const DetailHeader = withCurrentWorkspace()(
         {errors && <div className='error-messages'>
           <ValidationError>
             {summarizeErrors(errors && (
-              (filterTab === 'age' && ((ageMin && errors.ageMin) || (ageMax && errors.ageMax))) ||
-              (filterTab === 'date' && ((dateMin && errors.dateMin) || (dateMax && errors.dateMax)))
+              (filterTab === 'Age' && ((ageMin && errors.ageMin) || (ageMax && errors.ageMax))) ||
```

Comment:
The repetition of bare strings that have functional importance is a bit of a code smell. I'd lean towards defining 'Date' and 'Age' as constant strings at the top level, e.g. const AGE_FILTER_KEY = 'Age';

Reply:
Good idea, done.